docs/contributor/contribute-docs.md

```diff
@@ -16,6 +20,27 @@ the `karmada-io/website` repository.
 - For our docs we use markdown. If you are unfamiliar with Markdown, please see https://guides.github.com/features/mastering-markdown/ or https://www.markdownguide.org/ if you are looking for something more substantial. 
 - We get some additions through [Docusaurus 2](https://docusaurus.io/), a model static website generator.
 
+
+## How to Contribute
+
+If you want to contribute to Karmada, you can follow these steps:
```

Comment:
The current document focuses on `docs` contributing, but the added thing seems like [the common process](https://github.com/karmada-io/karmada/blob/master/CONTRIBUTING.md).

I don't think we should do it here.

Reply:
Removed How-to section from here